### ISSUE TITLE
v0.5.0 manylinux aarch64 wheels are missing on PyPI

### ISSUE THREAD

**Issue by Rongronggg9** [cher-nov/cryptg#43]
https://pypi.org/project/cryptg/0.5.0/#files

I noticed that the [corresponding build job](https://github.com/cher-nov/cryptg/actions/runs/11307214018/job/31448690426) was canceled manually. Was there anything wrong?

**Comment by Rongronggg9** [comment]
Oops, I overlooked the error. Thanks for the clarification.

I randomly looked into two time-out build jobs. They were both stuck at the same stage ("Compiling pyo3-macros v0.22.3"). I have no idea about the weird phenomenon.

BTW, `pyo3` v0.22.3 (`cryptg` v0.5.0 depends on it strictly) has been yanked and I noticed that #41 is merged to bump it to v0.22.4[^1]. Does it deserve a "post" release (i.e., v0.5.0.post0)? Since the failure has disappeared in recent workflow runs, such a release may succeed in building.

[^1]: `pyo3` v0.22.5 has been released after that.
